v2/internal/controllers/crd_apimanagement_20230501preview_test.go

```diff
@@ -38,7 +38,7 @@ func Test_ApiManagement_20230501preview_CRUD(t *testing.T) {
 	// Create an APIM instance. APIM has a soft delete feature; if you find that you
 	// hit this problem add the `restore` back in to resurrect it
 	service := apim.Service{
-		ObjectMeta: tc.MakeObjectMetaWithName(tc.Namer.GenerateName("apimv2")),
+		ObjectMeta: tc.MakeObjectMetaWithName(tc.Namer.GenerateName("apimanagementv2")),
```

Comment:
Had to update these since the resource with generated name already existed somewhere(not in our sub).